charts/sck-otel/templates/_config-Agent.tpl

```diff
@@ -531,4 +544,19 @@ service:
         - splunk_hec/platformMetrics
         {{- end }}
     {{- end }}
+    {{- if .Values.journaldLogs.enabled }}
+    logs/journald:
+      receivers:
+        - journald
+      processors:
+        - batch
+        - resource/splunk
```

Comment:
this `resource/splunk` processor doesn't fit well to journald logs. lets remove from journald processors.